tests/tamr_client/fake.py

```diff
@@ -27,7 +47,28 @@ def _to_kwargs(fake):
         path = req.get("path")
         url = "http://localhost/api/versioned/v1/" + path
 
-    return dict(method=req["method"], url=url, **resp)
+    # Get response body from either ndjson or json
+    if resp.get("ndjson") is not None:
+        resp["body"] = "\n".join((dumps(line) for line in resp["ndjson"]))
+    elif resp.get("json") is not None:
+        resp["body"] = dumps(resp["json"])
+
+    # Get expected request body from ndjson
+    if req.get("ndjson") is not None:
+        req["body"] = [x for x in req["ndjson"]]
```

Comment:
Unnecessary list comprehension:

```suggestion
        req["body"] = req["ndjson"]
```